src/components/DayBox.tsx

```diff
@@ -1,32 +1,23 @@
 import React from "react";
 import styled from "styled-components";
 
-const StyledCheckedBox = styled.div`
-  background-color: #d0142c;
-  border: 2px solid black;
-  border-left: 0rem solid;
-  color: white;
-  width: 2.5rem;
-  height: 2.5rem;
-  text-align: center;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  font-size: 1rem;
-`;
-
-const StyledUncheckedBox = styled.div`
-  border: 2px solid black;
+const BaseBox = styled.div`
+  border: 1px solid black;
   border-left: 0rem solid;
   color: black;
-  width: 2.5rem;
-  height: 2.5rem;
+  width: 100%;
```

Comment:
Could we actually make the day box a little smaller? It looks huge rn

Reply:
i was following Figma, and personally I think it looks okay. But I'll make it a bit smaller.